src/main/java/stream/flarebot/flarebot/util/votes/VoteUtil.java

```diff
@@ -0,0 +1,95 @@
+package stream.flarebot.flarebot.util.votes;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Guild;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import stream.flarebot.flarebot.scheduler.FlareBotTask;
+import stream.flarebot.flarebot.scheduler.Scheduler;
+import stream.flarebot.flarebot.util.ColorUtils;
+import stream.flarebot.flarebot.util.MessageUtils;
+import stream.flarebot.flarebot.util.buttons.ButtonUtil;
+import stream.flarebot.flarebot.util.objects.ButtonGroup;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class VoteUtil {
+
+    private static Map<String, VoteGroup> groupMap = new ConcurrentHashMap<>();
+    private static Map<String, VoteGroup.VoteRunnable> runnableMap = new ConcurrentHashMap<>();
+
+    public static void sendVoteMessage(UUID id, VoteGroup.VoteRunnable voteRunnable, VoteGroup group, long timeout, TextChannel channel, User user, ButtonGroup.Button... optionalButtons) {
+        EmbedBuilder votesEmbed = new EmbedBuilder();
+        votesEmbed.setDescription("Vote to " + group.getMessageDesc());
+        votesEmbed.addField("Yes Votes", "0", true);
```

Comment:
I still maintain this should be "Yes"and "No". Repeating vote is unneeded

(Also use damn chaining)